api/mesh/v1alpha1/rate_limit.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package kuma.mesh.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/mesh/v1alpha1";
+
+import "mesh/v1alpha1/selector.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+import "config.proto";
+
+message RateLimit {
+  // List of selectors to match dataplanes that rate limit will be applied for
+  repeated Selector sources = 1 [ (validate.rules).repeated .min_items = 1 ];
+
+  // List of selectors to match services that need to be rate limited.
+  repeated Selector destinations = 2
+      [ (validate.rules).repeated .min_items = 1 ];
+
+  message Conf {
+    message Http {
+      // The number of HTTP requests this RateLimiter allows
+      // +required
+      google.protobuf.UInt32Value requests = 1;
```

Comment:
I think this can be just uint32, we don't need to differentiate between nil and 0 value